sys/flash_map/src/flash_map.c

```diff
@@ -131,6 +132,40 @@ flash_area_align(const struct flash_area *fa)
     return hal_flash_align(fa->fa_device_id);
 }
 
+int
+flash_area_is_empty(const struct flash_area *fa, bool *empty)
+{
+    uint32_t data[64 >> 2];
+    uint32_t data_off = 0;
+    int8_t bytes_to_read;
+    uint8_t i;
+    int rc;
+
+    while(data_off < fa->fa_size)
+    {
```

Comment:
Can you fix the coding style?
See https://github.com/apache/incubator-mynewt-core/blob/master/CODING_STANDARDS.md